contracts/interfaces/IDimoRegistry.sol

```diff
@@ -6,6 +6,8 @@ pragma solidity ^0.8.13;
 interface IDimoRegistry {
     function setManufacturerMinted(address addr) external;
 
+    function updateIntegrationMinted(address from, address to) external;
```

Comment:
It's odd to have `setManufacturerMinted` and `updateIntegrationMinted`, no?

Reply:
I was supposed to update the manufacturer function too.